app/uk/gov/hmrc/helptosave/controllers/HelpToSaveAuth.scala

```diff
@@ -46,8 +48,17 @@ class HelpToSaveAuth(htsAuthConnector: AuthConnector) extends BaseController wit
   override def authConnector: AuthConnector = htsAuthConnector
 
   private type HtsAction = Request[AnyContent] ⇒ Future[Result]
+  private type HtsActionTyped[A] = Request[AnyContent] ⇒ A ⇒ Future[Result]
   private type HtsActionWithNino = Request[AnyContent] ⇒ NINO ⇒ Future[Result]
 
+  val authProviders: AuthProviders = AuthProviders(GovernmentGateway, PrivilegedApplication)
+
+  def authorised[A](retrievals: Retrieval[A])(action: HtsActionTyped[A])(implicit ec: ExecutionContext): Action[AnyContent] =
```

Comment:
perhaps someting like `ggOrStrideAuthorised` would be more descriptive here?

Reply:
This isn't actually being used anymore so I have removed it.